src/demos/calcite-tile-select.html

```diff
@@ -645,5 +559,91 @@
         </calcite-tile-select-group>
       </div>
     </div>
+
+    <!-- A horizonal line-->
+    <hr />
```

Comment:
Can this be split into a separate PR? It's technically not related to snapshot coverage. 

I should add that no action should be needed in this PR for the separated-out code/PRs after these are installed and `master` is merged into this one.

Reply:
I'll revert this.  I moved the `calcite-tile` examples down to the bottom since they're not the main focus of the demo, but this isn't important for the goal of this branch.